graph/src/blockchain/substreams_block_stream.rs

```diff
@@ -308,7 +308,7 @@ async fn process_substreams_response<C: Blockchain, F: BlockStreamMapper<C>>(
     match mapper
         .to_block_stream_event(logger, response.message, log_data)
         .await
-        .context("Mapping message to BlockStreamEvent failed")?
+        .map_err(BlockStreamError::from)?
```

Comment:
Context was turning this error into a anyhow Error which was in turn then not showing the full details of the underlying error